addons/mail/models/mail_notification.py

```diff
@@ -124,5 +124,5 @@ def _to_store_defaults(self):
             "mail_message_id",
             "notification_status",
             "notification_type",
-            Store.One("res_partner_id", ["name"], rename="persona"),
+            Store.One("res_partner_id", ["name", "email"], rename="persona"),
```

Comment:
If it is effectively the 'email' field, it can be formatted. You should use 'email_normalized', which holds the normalized version of email (of False if the email is missing / wrong, in which case we probably don't want to display it anyway).

Reply:
Indeed showing `name (email`) together is a mistake, should be either `email` or `name (email_normalized)`.
Looks like this is consistently a problem in Discuss code except with the improvements on recent chatter composer improvements you made with `RecipientsInput` :)

Reply:
So should we replace all occurrences of partner `email` by `email_normalized` in our code, as we really mean the latter?

Reply:
> So should we replace all occurrences of partner `email` by `email_normalized` in our code, as we really mean the latter?

I'd say all of them but recipient list (and recipients input and so on)

Reply:
Just note that a "wrong" email leads to a void email_normalized (aka: 'Raoul' with email 'caca', not sure how you prefer to display it - you can anyway do email = email_normalized or email).